testing/repository.go

```diff
@@ -0,0 +1,13 @@
+package vtesting
+
+import (
+	"github.com/Shopify/voucher/repository"
+)
+
+// Repository contains repository information pertaining to a repository
+type Repository struct {
+	org   repository.Organization
```

Comment:
Since we're using this for testing, could we export these fields?

```suggestion
	Organization   repository.Organization
```

Reply:
Done. But I kept the variable name as Org, so that it is different from the variable type. Or should it be called Organization?